node-admin/src/test/java/com/yahoo/vespa/hosted/node/admin/task/util/editor/StringEditorTest.java

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+package com.yahoo.vespa.hosted.node.admin.task.util.editor;
+
+import org.junit.Test;
+
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class StringEditorTest {
+    private final StringEditor editor = new StringEditor();
+    private final Cursor cursor = editor.cursor();
+
+    @Test
+    public void testBasics() {
+        assertCursor(0, 0, "");
+
+        cursor.write("hello");
+        assertCursor(0, 5, "hello");
+
+        cursor.write("one\ntwo");
+        assertCursor(1, 3, "helloone\ntwo");
+
+        cursor.deleteAll();
+        assertCursor(0, 0, "");
+
+        cursor.moveForward();
+        assertCursor(0, 0, "");
+
+        cursor.writeLine("foo");
+        assertCursor(1, 0, "foo\n");
```

Comment:
Perhaps you should also test writes and deletes in the interior of the buffer as part of the most basic stuff. I see it's tester further down, but only as part of more complex stuff.

Reply:
Ah, I see this is also tested in the TestBufferImplTest. Then it's not much of a point.